dbdialog/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -856,6 +862,17 @@ private void setInfo( DatabaseMeta meta ) {
       preserveReservedCaseCheck.setChecked( meta.preserveReservedCase() );
     }
 
+    if ( strictBigNumberInterpretaion != null ) {
+      //check if Oracle
+      if ( meta.getDatabaseInterface() instanceof OracleDatabaseMeta ) {
+        strictBigNumberInterpretaion.setVisible( true );
+        strictBigNumberInterpretaion.setChecked( meta.strictBigNumberInterpretation() );
```

Comment:
At this point we do not know whether meta is Oracle or not. Xul checkboxes in advanced options tab are defined in https://github.com/pentaho/pentaho-kettle/blob/master/dbdialog/src/main/resources/org/pentaho/ui/database/databasedialog.xul#L93-L109 and are same for all types of connections. So checking the "strictBigNumberInterpretaion" for doesn't  guarantee that the meta is Oracle.